scapy/main.py

```diff
@@ -46,10 +46,12 @@
     ("Craft packets like I craft my beer.", "Jean De Clerck"),
     ("Craft packets before they craft you.", "Socrate"),
     ("Craft me if you can.", "IPv6 layer"),
-    ("To craft a packet, you have to be a packet, and learn how to swim in the "  # noqa: E501
-     "wires and in the waves.", "Jean-Claude Van Damme"),
+    ("To craft a packet, you have to be a packet, and learn how to swim in "
+     "the wires and in the waves.", "Jean-Claude Van Damme"),
     ("We are in France, we say Skappee. OK? Merci.", "Sebastien Chabal"),
-    ("Wanna support scapy? Rate it on sectools! http://sectools.org/tool/scapy/", "Satoshi Nakamoto"),  # noqa: E501
+    ("Wanna support scapy? Rate it on sectools! "
+     "http://sectools.org/tool/scapy/", "Satoshi Nakamoto"),
+    ("What is dead may never die!", "Python 2"),
```

Comment:
Cool quote. I wonder how soon somehow will open an issue to drop Python3 support =)